setup.cfg

```diff
@@ -1,3 +1,4 @@
 [flake8]
 exclude = examples
+per-file-ignores = */api.py:F401
 ignore = E266,F401,E501,W503,E722,F821,E401,E231,W605,F841,E203,E713,E741,F403,E721,E302
```

Comment:
Not getting rid of `F401` from the `ignore` list just yet, because there are some other PRs that are needed to provide a complete fix.